omnidocs/tasks/layout_analysis/extractors/rtdetr.py

```
@@ -8,15 +8,20 @@
 import torch
 import torchvision.transforms as T
 from huggingface_hub import snapshot_download
+from transformers import RTDetrForObjectDetection, RTDetrImageProcessor
```

Ruff (0.12.2)

`transformers.RTDetrForObjectDetection` imported but unused

Remove unused import

(F401)